src/test.ts

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
```

Comment:
I guess that this file is not needed anymore as tests are handled by `jest`

Reply:
yes, don't need it anymore